### ISSUE TITLE
Improve documentation about including custom fonts (HTTP vs HTTPS)

### ISSUE THREAD

**Issue by sebastianbenz** [ampproject/amphtml#525]
According to the [documentation](https://github.com/ampproject/amphtml/blob/master/spec/amp-html-format.md#custom-fonts) states that custom fonts can be included via HTTP or HTTPS: 

> Authors are free to include all custom fonts via a @font-face CSS instruction via their custom CSS. Fonts included via @font-face must be fetched via HTTP or HTTPS scheme.

However, custom fonts are not shown in the preview if they are not served via HTTPS (at least if the fonts are served from a different domain). 

It would be great if we could make this more explicit in the documentation.

**Comment by dvoytenko** [comment]
Sebastian let's confirm exactly what's going on there. It sounded like this was actually a bug?

**Comment by cramforce** [comment]
Yeah, seems like a bug. HTTTP should not be required for fonts.

**Comment by sebastianbenz** [comment]
Ok - I will leave it open until it's fixed.

**Comment by dvoytenko** [comment]
@powdercloud could you please confirm?

**Comment by powdercloud** [comment]
We don't do anything to disallow a @font-face thus far. The relevant part is l. 467 here:
https://github.com/ampproject/amphtml/blob/master/validator/validator.protoascii
In case you're wondering the instruction above that in l. 448 is necessary to allow font-face in the CSS parser, so it will parse it but other than CSS syntax validation not do anything to it.
For the font tag (the a), the fonts can only come from a couple whitelisted sources. See l. 123 here:
https://github.com/ampproject/amphtml/blob/master/validator/validator.protoascii
So basically: I think we need some example of what's not working, e.g. a specific doc at a URL. While a snippet of HTML is OK to figure out what's up with the validator, it is possible that what's going on here is something else. The validator never changes the documents (it's not a sanitizer) but perhaps other parts of our serving system do, so we could look into that with a URL.

**Comment by dvoytenko** [comment]
@sebastianbenz Sebastian, could you please give us a URL to reproduce the problem?
